site/docs/build-ref.html

```diff
@@ -348,7 +348,8 @@ <h4>Package names, <code>//<b>package-name</b>:...</code></h4>
 </p>
 
 <p>
-  Although Bazel allows a package at the build root (e.g. <code>//:foo</code>), this
+  Although Bazel allows a package at the build root (e.g.
```

Comment:
It don't think this change is better.  It talks about packages, but the code block is a target.
... _package_ at the build route ...  defines a _target_ ...
It conflates the two.

Reply:
I disagree that it conflates anything but leaving that aside, I hope you can see that a standard English interpretation of what's there appears to be saying that `//:foo` is the name of a package.  

And I understand why the current text uses a target name; if you wanted to use the package name, you'd have to write

```suggestion
  Although Bazel allows a package at the build root (i.e. <code>//</code>), this
```

which is confusing in that `<code>//</code>` reads as though it refers to the build root, not the package, and the package in question has an empty name.  I'm guessing closer to the truth would be that the package at the build root always exists, we are discouraged from putting targets in it:

```suggestion
  Although Bazel allows targets at the build root (i.e. <code>//:foo</code>), it
  is discouraged: projects should attempt to only create targets in packages
  with descriptive names.
```